blog-investigation/src/post/postApiService.js

```diff
@@ -19,4 +19,38 @@ export default class PostApiService {
       },
     });
   }
+
+  static createPost(blogId, model) {
+    const token = localStorage.getItem("token");
```

Comment:
please have a look at this article: Using Axios interceptors
[https://blog.logrocket.com/using-axios-set-request-headers/](https://blog.logrocket.com/using-axios-set-request-headers/)

Reply:
please create separate task for it